Scripts/BeforeBuild.ps1

```diff
@@ -1,19 +1,5 @@
-git clone -q --branch=master https://github.com/Microsoft/vcpkg.git vcpkg
-if (Test-Path -Path 'cache\vcpkg-installed') {
-	New-Item -Type dir vcpkg\installed
-	Copy-Item cache\vcpkg-installed\* vcpkg\installed -recurse
-}
-Copy-Item hack\portfile.cmake vcpkg\ports\libwebsockets\portfile.cmake -Force
-pwd
-.\vcpkg\bootstrap-vcpkg.bat
-.\vcpkg\vcpkg integrate install
-.\vcpkg\vcpkg install libwebsockets
-pwd
-if (Test-Path -Path 'cache\vcpkg-installed') {
-	Remove-Item cache\vcpkg-installed -Force -Recurse
-}
-New-Item -Type dir cache\vcpkg-installed
-Copy-Item vcpkg\installed\* cache\vcpkg-installed -Recurse -Force
+Copy-Item hack\portfile.cmake C:\Tools\vcpkg\ports\libwebsockets\portfile.cmake -Force
+vcpkg install libwebsockets

```

Comment:
We'd (vcpkg) be happy to accept a PR to update libwebsockets to 2.2.1 so you don't need to have this hack anymore! 😄

Reply:
I don't use libwebsockets, so I won't make PR for that. Can't handle any possible issue.
@robertabcd should do this.

Reply:
@ras0219-msft I can't guarantee my time to make a PR. One thing I'm nervous about is the UWP patch here:
https://github.com/Microsoft/vcpkg/blob/master/ports/libwebsockets/0001-Fix-UWP.patch
I can't be sure upgrading would not break UWP.